.travis.yml

```diff
@@ -10,48 +10,109 @@ env:
+       
+     before_deploy:
+       - docker login -u="${QUAY_ID}" -p="${QUAY_TOKEN}" quay.io
+       
+     deploy:
+       # Deploy alpha builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
 
-before_deploy:
-  - docker login -u="${QUAY_ID}" -p="${QUAY_TOKEN}" quay.io
+       # Deploy released builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
+           
+   - stage: build and push multi-arch image
+     arch: amd64
+     script:
+       - export DOCKER_CLI_EXPERIMENTAL=enabled    
+       - if [ -n "${TRAVIS_TAG}" ]; then docker manifest create quay.io/razee/remoteresources3:${TRAVIS_TAG} quay.io/razee/remoteresources3:${TRAVIS_TAG}-amd64 quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le; fi
+      
+     before_deploy:
+       - docker login -u="${QUAY_ID}" -p="${QUAY_TOKEN}" quay.io
+       
+     deploy:
+       # Deploy alpha builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
 
-deploy:
-  # Deploy alpha builds
-  - provider: script
-    script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}"
-    skip_cleanup: true
-    on:
-      tags: true
-      condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
-
-  # Deploy released builds
-  - provider: script
-    script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}"
-    skip_cleanup: true
-    on:
-      tags: true
-      condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
-  - provider: releases
-    file: /tmp/resource.yaml
-    skip_cleanup: true
-    api_key: "${GITHUB_TOKEN}"
-    on:
-      tags: true
-      condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
-  - provider: npm
-    email: "${NPMJS_EMAIL}"
-    api_key: "${NPMJS_API_KEY}"
-    name: "${TRAVIS_TAG}"
-    skip_cleanup: true
-    on:
-      tags: true
-      condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
+       # Deploy released builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
+       - provider: releases
+         file: /tmp/resource.yaml
+         skip_cleanup: true
+         api_key: "${GITHUB_TOKEN}"
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
+       - provider: npm
```

Comment:
line 24 in the 1st stage modifies the package.json with the new version to be published. Does that change carry over to this 3rd stage?